docs/installing.md

```diff
@@ -0,0 +1,61 @@
+# Installing
+
+In most cases all you need to do is
+```bash
+pip install ipympl
+```
+or alternatively with `mamba`/`conda`
+
+```bash
+mamba install -c conda-forge ipympl
+```
+
+and then restart Jupyter.
+
+
+### Jupyterlab < 3
+If you use JupyterLab 2, you still need to install the labextension manually:
+
+```bash
+conda install -c conda-forge nodejs
+jupyter labextension install @jupyter-widgets/jupyterlab-manager jupyter-matplotlib
+```
+
+
+<!--
+## Sagemath CoCalc
+TODO
+-->
+
+### Google Colab
+
+To use `ipympl` in colab follow the instructions [here](https://colab.research.google.com/github/googlecolab/colab-cdn-widget-manager/blob/main/notebooks/ipympl.ipynb)
+
```

Comment:
TODO:

add:

```python
from google.colab import output
output.enable_custom_widget_manager()
```

does that work? I guess the linked notebook here is out of date?

Reply:
> I guess the linked notebook here is out of date?

It looks like it's outdated indeed. There have been multiple changes in the Colab API before it was working properly.

I believe it works nicely now with the code snippet I shared :) At least it worked like a charm the other day when I tried ipycanvas (which is similar in some ways to ipympl)

Reply:
done :)